src/lib/viewers/controls/color-picker/ColorPickerPalette.tsx

```diff
@@ -1,13 +1,14 @@
 import React from 'react';
+import { AnnotationColor } from '../../../AnnotationModule';
 import './ColorPickerPalette.scss';
 
+const colors = Object.values(AnnotationColor);
+
 export type Props = {
-    onColorSelect: (color: string) => void;
+    onColorSelect: (color: AnnotationColor) => void;
```

Comment:
In the same vein of keeping the color picker palette annotations agnostic, could the colors be passed in as a prop?

Reply:
Yes, that sounds good.